src/fairchem/core/common/utils.py

```diff
@@ -1231,11 +1231,23 @@ def scatter_det(*args, **kwargs):
     return out
 
 
-def get_commit_hash():
+def get_commit_hash() -> str:
+    core_hash = get_commit_hash_for_repo(fairchem.core.__path__[0])
+    experimental_hash = None
+    try:
+        experimental_hash = get_commit_hash_for_repo(fairchem.experimental.__path__[0])
+        return f"core:{core_hash},experimental:{experimental_hash}"
+    except (NameError, AttributeError):
+        return f"core:{core_hash},experimental:NA"
+
+
+def get_commit_hash_for_repo(
+    git_repo_path: str,
+) -> Optional[str]:  # noqa: UP007 python 3.9 requires Optional still
```

Comment:
better to use "from __future___ import annotations" to use py 3.10 + annotations and get rid of Optional and the lint ignore comment

Reply:
ah! I didn't realize this, i will give this a shot!